openid-4-verifiable-presentations-1_0.md

```diff
@@ -2454,57 +2284,46 @@ AnonCreds is a Credential format defined as part of the Hyperledger Indy project
 
 To be able to request AnonCreds, there needs to be a set of identifiers for Verifiable Credentials, Verifiable Presentations ("proofs" in Indy terminology) and crypto schemes.
 
-Credential format identifier is `ac_vc` for a Credential, and `ac_vp` for a Presentation.
-
-Identifier for a CL-signature crypto scheme used in the examples in this section is `CLSignature2019`.
-
-### Example Credential
-
-The following is a non-normative example of an AnonCred Credential that will be used throughout this section. 
+The Credential format identifier is `ac_vp` to request a Presentation.
 
-<{{examples/credentials/ac_vc.json}}
+The identifier for the CL-signature crypto scheme used in the examples in this section is `CLSignature2019`.
 
-The most important parts for the purpose of this section are `scheme_id` parameter and `values` parameter that contains the actual End-User claims. 
+### DCQL
 
-### Presentation Request 
+#### Parameters in the `meta` parameter in Credential Query {#anoncreds_meta_parameter}
 
-#### Request Example {#anoncreds_request}
+The following are AnonCreds specific parameters in the `meta` parameter in a Credential Query as defined in (#credential_query):
 
-The following is a non-normative example of an Authorization Request:
+`schema_id_values`:
+: OPTIONAL. An array of strings that specifies the allowed values for the `schema_id` of the requested Verifiable Credential. It MUST be a valid scheme identifier as defined in [@Hyperledger.Indy].
 
-<{{examples/request/request.txt}}
+`cred_def_id_values`:
+: OPTIONAL. An array of strings that specifies the allowed values for the `cred_def_id` of the requested Verifiable Credential. It MUST be a valid credential definition identifier as defined in [@Hyperledger.Indy].
 
-The following is a non-normative example of the content of the `presentation_definition` parameter:
+`proof_type_values`:
+: OPTIONAL. An array of strings that specifies the types of proofs that the Verifier accepts to be used in the Verifiable Presentation, for example `CLSignature2019`.
```

Comment:
I would not add this TBH. AnonCreds only supports CL signatures, I've never seen a deployment that does not CL signatures. There is effort ongoing for an AnonCreds v2, but this has a different model than AnonCreds v1,  so it should have a different format identifier IMO. So just the `ac_vp` identifier should be enough

Reply:
Thanks, I will remove it.